### PR TITLE
Team Augmentation Page

### PR DESCRIPTION
**Pull request by wadie**
Adds the team augmentation page, following our agreed-upon `services` template.
Let me know if any changes are necessary.

### CODE REVIEW

File: src/ui/components/TeamAugmentation/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@block grid-block from "../../styles/blocks/grid-block.block.css";
+
+:scope {
+  block-name: service;
+  extends: screen;
+}
+
+.display {
+  margin-bottom: 1rem;
+}
```
Comment (marcoow) [inline src/ui/components/TeamAugmentation/stylesheet.css]: Is this used anywhere?

File: src/ui/components/TeamAugmentation/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+.featured-right-background {
+  background-color: resolve("shape-right-lg.background");
+  background-color: #007DF6;
+}
```
Comment (marcoow) [inline src/ui/components/TeamAugmentation/stylesheet.css]: This does not seem to be used?

File: src/ui/components/TeamAugmentation/stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+.featured-left-background{
+  background-color: resolve("shape-left-md.background");
+  background-color: #007DF6;
+}
```
Comment (marcoow) [inline src/ui/components/TeamAugmentation/stylesheet.css]: This does not seem to be used as well?

File: src/ui/components/TeamAugmentation/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+.featured-center {
+  justify-content: resolve('shape.shape-center');
+  justify-content: flex-start;
+}
```
Comment (marcoow) [inline src/ui/components/TeamAugmentation/stylesheet.css]: …neither does this one

Comment (ghislaineguerin) [reply src/ui/components/TeamAugmentation/stylesheet.css]: @wadie @marcoow we can delete all unused styles.

**Comment by wadie** [comment]
Just removed all these rules from every services component.

**Comment by ghislaineguerin** [comment]
@wadie I think we are no longer doing the lowercase sentences. It should all be sentence case. Other than that it looks good.

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
@wadie please don't merge before the checks have passed - now we don't know whether the checks would have been successful and if they had not been we have no broken `master` (or `new-simplabs` in this case).

**Pull request merged**